manifest.json

```diff
@@ -431,9 +431,9 @@
         "targetDirPrefix": "ui-map-"
       },
       "uiSortable": {
-        "version": "0.12.6",
+        "version": "0.17.1",
```

Comment:
Just to check, you are actually using this when testing, right? (I think it gets generated into third_party.js by the build scripts, but I'm not sure when exactly that generation happens. You might want to double-check, by inspecting the source files in your local browser, that the JS that's serving includes some strings from the v0.17.1 file that don't exist in the v0.12.6 file.

Reply:
Yes I checked this. The generation happens every time you run start.sh I think(it has a generate step). I did a ctrl+f on the third_party.js file and it is a successful search for the 0.17.1 version